doc/classes/ItemList.xml

```diff
@@ -387,6 +381,9 @@
 		<member name="icon_scale" type="float" setter="set_icon_scale" getter="get_icon_scale" default="1.0">
 			The scale of icon applied after [member fixed_icon_size] and transposing takes effect.
 		</member>
+		<member name="items_count" type="int" setter="set_item_count" getter="get_item_count" default="0">
```

Comment:
This should likely be `item_count` for the sake of sounding natural to English speakers.

Reply:
Came here to say the same, although it's just a nitpick. It's consistent with the method names, too.

Reply:
I agree too, PR welcome :)